src/tasks/peripherals/task_nfc.c

```diff
@@ -23,7 +24,7 @@ ruuvi_driver_status_t task_nfc_init (void)
     memcpy (version_string, fw_prefix, sizeof (fw_prefix));
     written = snprintf ( (char *) (version_string + sizeof (fw_prefix)),
                          APPLICATION_COMMUNICATION_NFC_TEXT_BUFFER_SIZE - sizeof (fw_prefix),
-                         "%s", APPLICATION_FW_VERSION);
+                         "%s", APPLICATION_FW_VERSION);  // Taken from compile flag. May be MM.nn.rr or git hash
```

Comment:
The actual source is in application_config.h, which happens to pick it from compiler flag or use a default value "RuuviFW DEVEL" as of right now. 
This comment would be better if it pointed to the application_config.h where the macro is documented, if it is documented twice the comments are going to end up out of sync.